docs/openstack/ceph_backend_configuration.md

```diff
@@ -19,7 +19,18 @@ CEPH_KEY=$($CEPH_SSH "cat /etc/ceph/ceph.client.openstack.keyring | base64 -w 0"
 CEPH_CONF=$($CEPH_SSH "cat /etc/ceph/ceph.conf | base64 -w 0")
 ```
 
-## Procedure - Ceph backend configuration
+## Modify capabilities of the "openstack" user to accommodate Manila
+
+On TripleO environments, the CephFS driver in Manila is configured to use 
+its own keypair. For convenience, let's modify the `openstack` user so that we 
+can use it across all OpenStack services.
+
+```
+$CEPH_SSH cephadm shell
+ceph auth caps client.openstack mgr 'allow *' mon 'allow r, profile rbd' osd 'profile rbd pool=vms, profile rbd pool=volumes, profile rbd pool=images, allow rw pool manila_data'
```

Comment:
I've never seen it happen, and `manila_data` is a good default here, but I'm wondering if we want to inspect the `Ceph cluster` (e.g. `cephadm shell -- ceph osd pool ls`) and double check the `Manila` pools [1]  before adding the missing `allow rw pool manila_data` (is `manila_metadata` missing or do we need it anymore?).

Also, I'm wondering if the command on L30 works for you. I had troubles trying to update an existing keyring and I ended up doing the following:

```
cephadm shell -m /etc/ceph
```

Edit the keyring located in  `/mnt/ceph.client.openstack.keyring`
and add the missing caps (`profile rbd pool=manila_data, profile rbd pool=manila_metadata`)

The file will result in the following:

```
[client.openstack]
        key = <key>
        caps mgr = "allow *"
        caps mon = "profile rbd"
        caps osd = "profile rbd pool=vms, profile rbd pool=volumes, profile rbd pool=images, profile rbd pool=manila_data, profile rbd pool=manila_metadata"
```

Update the keyring in ceph (reimport the same key):

```
ceph auth rm client.openstack
ceph auth import -i /mnt/ceph.client.openstack.keyring
```

[1] https://github.com/openstack/tripleo-heat-templates/blob/master/deployment/cephadm/ceph-base.yaml#L549-L551

Reply:
nevermind, I tried the command you mentioned and it works as expected (I probably messed up w/ my previous environment).
Resolving the comment as I tested and it works